v2/brigadier/src/jobs.ts

```diff
@@ -1,15 +1,56 @@
 import { Event } from "./events"
+import { ConcurrentGroup, SerialGroup } from "./groups"
+import { Runnable } from "./runnables"
 
 const defaultTimeout: number = 1000 * 60 * 15
 
-export class Job {
+/**
+ * A Brigade job.
+ * 
+ * Instances of Job represent containers that your Brigade script can
+ * run using the Job#run method.
+ * 
+ * The Job#primaryContainer, initialised from the constructor image argument, determines
+ * how long the job runs and whether it is considered successful.By default, the container is simply
+ * executed (via its default entry point). Set the Job#primaryContainer#command
+ * property to run a specific command in the container instead.
+ * Other containers, specified via Job#sidecarContainers, are automatically
+ * terminated after the primary container completes.
```

Comment:
There's a lag on this which is configurable by the operator who installs Brigade. It's not worth getting into the technical details of why, but it might be worth mentioning the delay in order to calibrate user expectations vis-a-vis "automatically terminated."

Reply:
Tweaked to "automatically terminated a short time after..." - does that meet your needs?  I don't want to get too much into the details in a JSDoc lest it turn into an essay...

Reply:
Yep... that's fine as long as we're not accidentally implying it's immediate.